katalog/prometheus-adapter/deployment.yaml

```
@@ -35,73 +35,73 @@ spec:
-        image: registry.k8s.io/prometheus-adapter/prometheus-adapter:v0.12.0
-        livenessProbe:
-          failureThreshold: 5
-          httpGet:
-            path: /livez
-            port: https
-            scheme: HTTPS
-          periodSeconds: 5
-        name: prometheus-adapter
-        ports:
-        - containerPort: 6443
-          name: https
-        readinessProbe:
-          failureThreshold: 5
-          httpGet:
-            path: /readyz
-            port: https
-            scheme: HTTPS
-          periodSeconds: 5
-        resources:
-          limits:
-            cpu: 250m
-            memory: 180Mi
-          requests:
-            cpu: 102m
-            memory: 180Mi
-        securityContext:
-          allowPrivilegeEscalation: false
-          capabilities:
-            drop:
-            - ALL
-          readOnlyRootFilesystem: true
-          runAsNonRoot: true
-          seccompProfile:
-            type: RuntimeDefault
-        startupProbe:
-          failureThreshold: 18
-          httpGet:
-            path: /livez
-            port: https
-            scheme: HTTPS
-          periodSeconds: 10
-        volumeMounts:
-        - mountPath: /tmp
-          name: tmpfs
-          readOnly: false
-        - mountPath: /var/run/serving-cert
-          name: volume-serving-cert
-          readOnly: false
-        - mountPath: /etc/adapter
-          name: config
-          readOnly: false
+        - args:
+            - --cert-dir=/var/run/serving-cert
+            - --config=/etc/adapter/config.yaml
+            - --metrics-relist-interval=1m
+            - --prometheus-url=http://prometheus-k8s.monitoring.svc:9090/
+            - --secure-port=6443
+            - --tls-cipher-suites=TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA256,TLS_ECDHE_ECDSA_WITH_AES_128_CBC_SHA,TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA,TLS_ECDHE_ECDSA_WITH_AES_256_CBC_SHA,TLS_RSA_WITH_AES_128_GCM_SHA256,TLS_RSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_128_CBC_SHA,TLS_RSA_WITH_AES_256_CBC_SHA
+          image: registry.k8s.io/prometheus-adapter/prometheus-adapter:v0.12.0
+          livenessProbe:
+            failureThreshold: 5
+            httpGet:
+              path: /livez
+              port: https
+              scheme: HTTPS
+            periodSeconds: 5
+          name: prometheus-adapter
+          ports:
+            - containerPort: 6443
+              name: https
+          readinessProbe:
+            failureThreshold: 5
+            httpGet:
+              path: /readyz
+              port: https
+              scheme: HTTPS
+            periodSeconds: 5
+          resources:
+            limits:
+              cpu: 250m
+              memory: 512Mi
```

we already have the patch via kustomize, let's leave this like upstream so we don't confuse the next maintainer
```suggestion
              memory: 180Mi
```